test/framework/module_generator.py

```diff
@@ -50,10 +50,10 @@
 
 
 class ModuleGeneratorTest(EnhancedTestCase):
-    """ testcase for ModuleGenerator """
+    """ testcase for ModuleGeneratorTcl """
```

Comment:
some tests will need to be made aware of whether they are being run with Tcl syntax enabled or Lua syntax enabled, i.e. the ones with hard checks on syntax

@pforai: you should be able to pull that off, right?

```python
if build_option('modules_syntax') == ModuleGeneratorTcl.SYNTAX:
    # hard checks on Tcl syntax
elif build_option('modules_syntax') == ModuleGeneratorLua.SYNTAX:
    # hard check on Lua syntax
else:
    self.assertTrue(False, "Using known module file syntax")
```